### ISSUE TITLE
Dropdowns in Cards get clipped

### ISSUE THREAD

**Issue by s-rusev** [vmware/clarity#755]
When using clr-dropdown in a clickable card's footer the dropdown goes behind the next card. Also when hovering on dropdown items the next card goes above the dropdown.

[x ] bug
[ ] feature request
[ ] enhancement

### Expected behavior
The dropdown should go over the next card

### Actual behavior
See attached plunkr - https://plnkr.co/edit/9xrEtr?p=preview
The dropdown goes behind the next card

### Reproduction of behavior
Click on the Actions dropdown
Note that if the card is not clickable - everything works as expected in the plnkr. 

(Although when I removed the clickable class in my app - the dropdown continued showing under the next card - it has to do something with the animations I'm using: )
<div class="card" [@flyInOut]="'in'">

animations: [  trigger('flyInOut', [
      state('in', style({transform: 'translateX(0)'})),
      transition('void => *', [
        style({transform: 'translateX(-100%)'}),
        animate(100),
      ]),
      transition('* => void', [
        animate(100, style({transform: 'translateX(100%)'}))
      ])    ])   ]
When I removed them the dropdown showed correctly.

### Environment details

* **Angular version:** 2.0.X

* **Clarity version:** latest

* **OS and version:**  Windows 8.1

* **Browser:** [ Chrome latest | Firefox latest | IE latest ] haven't tested with others

**Comment by adityarb88** [comment]
Hi @s-rusev: We do not recommend using dropdowns in Clickable cards. Clickable cards is intended to be interactive content (refer: https://www.w3.org/TR/html5/dom.html#interactive-content-0). Interactive content should not have a Interactive content descendant.

**Comment by s-rusev** [comment]
Then in my opinion it would be better if you remove the "Dropdowns in Cards" section from https://vmware.github.io/clarity/documentation/cards or write it there that it is not recommended, because I've ended up with the impression that it is perfectly fine to use dropdowns in cards. 
Also if I'm using a button group with overflow - like here https://vmware.github.io/clarity/documentation/button-group -> Overflow section I'll have the same problem. 

So what is your suggestion to "interact" with the card? If I have many actions in a card's footer how I should display them when obviously I have limited space and problem with the overflows? Thanks!

**Comment by adityarb88** [comment]
Hi @s-rusev: Yes. I will add a ticket to mention that clickable cards should behave like buttons or links and should not contain and other clickable elements inside of it. But you can still use dropdowns or button groups for that matter in cards which are not clickable. That should work fine.

**Comment by s-rusev** [comment]
The problem persists if I'm using the mentioned animations too (without the clickable class).  I can update the plnkr if you like.

**Comment by s-rusev** [comment]
Here is the updated plnkr - https://plnkr.co/edit/dG9ulB?p=preview

**Comment by adityarb88** [comment]
This does look like a valid issue. We will take a look at this. Thanks for reporting this!

**Comment by hippee-lee** [comment]
This is still an issue with popovers and the z-index. Have not prioritized a solution yet.

**Comment by gnomeontherun** [comment]
I reviewed this bug, the issue only exists when the animation is used on the card. Since the demo is no longer working I made a new one. 

https://stackblitz.com/edit/clarity-light-theme-v1-0-rxtk8f?file=src%2Fapp%2Fapp.component.ts

The issue is that translateX forces position relative on the card, leading to clipping. I'm going to mark this as wontfix since we cannot control this behavior and the application needs to determine a different way to avoid clipping and `position: relative` being applied.
